src/js/components/products/listItem.tsx

```diff
@@ -31,25 +31,25 @@ const ProductItem = ({ item }: { item: Product }) => {
           icon={<ProductIcon item={item} />}
           bodyClassName="slds-card__body_inner"
         >
-          <div>
-            <span className="slds-text-title slds-m-right_xx-small">
+          <div className="md-product-meta slds-grid slds-wrap slds-grid_vertical-align-center">
```

Comment:
What's the benefit of these grid classes? Before we had auto-truncation, but now it overflows:

Before:

![Screen Shot 2022-07-16 at 9 52 35 PM](https://user-images.githubusercontent.com/552316/179380868-ecb338aa-0c4e-4749-a354-857efdc072d9.jpg)

After:

![Screen Shot 2022-07-16 at 9 52 50 PM](https://user-images.githubusercontent.com/552316/179380871-4c9dbbfc-6330-41c6-9c78-38c3f993e2e4.jpg)

Reply:
It was to add better row gaps without adding more vertical margin and padding, but I didn't test with a longer tag name to see it removed truncation. I removed all of this and am going to be OK with the uneven alignment of the first row cards when there is a mixture of cards with tags and without. But once we get to row two, it doesn't matter anyway. 

I wonder if truncating the titles and maybe the descriptions makes sense if the tags aren't limited. I can see cutting the description off (maybe max-height) at some point... but these card layouts can look really strange if there are a lot of tags on some cards.

![Screen Shot 2022-07-19 at 10 47 38 AM](https://user-images.githubusercontent.com/1581694/179796572-bc0ed8f9-b94a-4768-b6d6-a93dcfe78d35.jpg)

I'm curious why the first card's description is truncated but not the second card. I am not going to dive into that one but I thought it was interesting. 

But here's is what started this in the first place, the unevenness of that second card  without tags.

![Screen Shot 2022-07-19 at 11 04 11 AM](https://user-images.githubusercontent.com/1581694/179797208-14618b10-8009-4271-8d09-be03c819b9e5.jpg)